django/db/backends/base/schema.py

```diff
@@ -314,6 +314,10 @@ def delete_model(self, model):
         self.execute(self.sql_delete_table % {
             "table": self.quote_name(model._meta.db_table),
         })
+        # Remove all deferred statements referring the deleted table.
```

Comment:
better word choice:
referring -> referencing
(or "referring to" would also be correct)